### ISSUE TITLE
Error message in a legit graphviz file

### ISSUE THREAD

**Issue by csrabak** [laingsimon/preview-dot#15]
I get : 
System.InvalidOperationException: Invalid image data returned:  ---> System.ArgumentException: Parâmetro inválido. _meaning "invalid parameter"_
   em System.Drawing.Image.FromStream(Stream stream, Boolean useEmbeddedColorManagement, Boolean validateImageData)
   em PreviewDot.PreviewHandlerForm.<_UpdatePreview>d__7.MoveNext()
   --- Fim do rastreamento de pilha de exceções internas --- _meaning end of internal exception stack tracking_
   em PreviewDot.PreviewHandlerForm.<_UpdatePreview>d__7.MoveNext()

The "culprit" file is attached
[culprit.txt](https://github.com/laingsimon/preview-dot/files/3567360/culprit.txt)

file was renamed as GiHub doesn't allow the extension .gv. . .

**Comment by laingsimon** [comment]
@csrabak The issue with this file is it contains a [BOM](https://en.wikipedia.org/wiki/Byte_order_mark) which dot.exe doesn't support

```
dot.exe -Tpng culprit.gv
Error: culprit.gv: syntax error in line 1 near '∩╗┐digraph'
```

The stream is verbatim sent to DOT.exe but could be manipulated to solve this issue. Leaving this issue open for further investigation.

For now the work around is to encode the file without the [UTF-8 BOM](https://en.wikipedia.org/wiki/Byte_order_mark).

The [attached file](https://github.com/laingsimon/preview-dot/files/3607272/issue-15-no-BOM.txt) demonstrates that this works without issue.

**Comment by laingsimon** [comment]
Resolved in release v0.3 - thanks for identifying the fault, a new installer is available for download.
